lib/rules/yoda.js

```diff
@@ -20,6 +20,15 @@ function isComparisonOperator(operator) {
 }
 
 /**
+ * Determines whether an operator is an equality operator.
+ * @param {String} operator The operator to check.
+ * @returns {boolean} Whether or not it is an equality operator.
+ */
+function isEqualityOperator(operator) {
+    return (/^(==|===)$/).test(operator);
```

Comment:
I don't think that's necessary. We can add `!=` later if there will be enough requests.